openedx/features/enterprise_support/README.rst

```diff
@@ -0,0 +1,13 @@
+Enterprise Support
+------------------
+
+This directory contains a Django application to support usage of
+enterprise features within edx-platform. The majority of the capabilities
+are provided through the external edx-enterprise library that can be found
+here: `https://github.com/edx/edx-enterprise`_.
+
+Note: this Django app is only necessary due to limitations in extending
+edx-platform through installed libraries. The eventual goal is to have
+all of this functionality provided through edx-enterprise. New code should
```

Comment:
Not exactly. Our huge, big, long-term goal is to actually implement Enterprise as an IDA - in which case, the stuff in Enterprise Support would likely just be REST clients for various tasks. We've been intentionally abstracting calls into edx-enterprise at this choke point so that when (if) that eventually happens, we have a single point that we can refactor without affecting the rest of the platform.